l10n_be_cooperator_national_number/models/res_partner_id_category.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, models
+
+
+class ResPartnerIdCategory(models.Model):
+    _inherit = "res.partner.id_category"
+
+    @api.model
+    def get_belgian_id_category(self):
+        return self.env.ref(
+            "l10n_be_partner_identification.l10n_be_national_registry_number_category"
+        )
```

Comment:
since this is something always used with a partner, wouldn’t it make more sense to define this on `res.partner` (or at least add another function there calling this one to avoid having to access `self.env["res.partner.id_category"]` each time)?